src/views/ReferralProgram.vue

```diff
@@ -141,17 +143,29 @@ export default class ReferralProgram extends Mixins(
   readonly LogoSize = LogoSize;
 
   @Getter('tokenXOR', { namespace: 'assets' }) tokenXOR!: AccountAsset;
-  @Getter('invitedUsers', { namespace }) invitedUsers!: Array<string>;
+  @Getter('invitedUsers', { namespace }) invitedUsers!: Nullable<Array<string>>;
 
-  @Action('getInvitedUsers', { namespace }) getInvitedUsers!: (referralId: string) => Promise<void>;
+  @Action('resetInvitedUsersSubscription', { namespace }) resetInvitedUsersSubscription!: AsyncVoidFn;
+  @Action('subscribeInvitedUsers', { namespace }) subscribeInvitedUsers!: (referrerId: string) => AsyncVoidFn;
+  @Action('unsubscribeInvitedUsers', { namespace }) unsubscribeInvitedUsers!: AsyncVoidFn;
   @Action('setBound', { namespace }) setBound!: (isBond: boolean) => Promise<void>;
 
+  @Watch('isLoggedIn')
+  @Watch('isSoraAccountConnected')
```

Comment:
`isLoggedIn `and `isSoraAccountConnected` is the same getter with different namings . Please leave one for tracking (`isSoraAccountConnected `from `WalletConnectMixin`)